tapdance/logger.go

```diff
@@ -35,3 +40,36 @@ func Logger() *logrus.Logger {
 	})
 	return logrusLogger
 }
+
+func StatsReporting(stats *pb.SessionStats) {
```

Comment:
There are lots of TODOs and commented out lines of code in this function.
Do we need all of them, or can we remove some?